theme/less/main.less

```diff
@@ -48,3 +48,34 @@ main
   padding: 15px;
   max-width: 1000px;
 }
+
+#breadcrumb
+{
+  font-size: 0.92em;
```

Comment:
Note: In the Drupal version using `em` does make difference, but on current version changing this to `rem` doesn't affect size, I would suggest to use `rem` instead and adjust sizes accordingly to visually look same comparing the computed size.

You probably know the difference between `em` and `rem`, but I would like to relate to that `rem` is a relative to the HTML element, while `em` to the closest parent. Having dependency on HTML in my experience make the size calculation simpler, rather than when the item is dependent on multiple parents. Sometimes it make sense to use `em` instead `rem`, but in this case I can see that it adds much of value.